src/Symfony/Component/Security/Http/EventListener/IsCsrfTokenValidAttributeListener.php

```diff
@@ -49,4 +56,18 @@ public static function getSubscribedEvents(): array
     {
         return [KernelEvents::CONTROLLER_ARGUMENTS => ['onKernelControllerArguments', 25]];
     }
+
+    private function getIsCsrfTokenValidId(string|Expression $id, Request $request, array $arguments): string
```

Comment:
I would name the method `getTokenId`

Reply:
I inspired from [IsGrantedAttributeListener](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php#L83)

Reply:
indeed should have been maybe `getIsCsrfTokenValidTokenId` to better match the inspiration.
so I did the change in a fixup commit